src/Tizen.Security.PrivacyPrivilegeManager/Tizen.Security/PrivacyPrivilegeManager.cs

```diff
@@ -203,23 +386,27 @@ internal ResponseContext(string privilege)
                 add
                 {
                     _ResponseFetched += value;
+                    if (!s_responseMap.ContainsKey(_privilege))
+                    {
+                        s_responseMap[_privilege] = new ResponseContext(_privilege);
```

Comment:
Why do you create a new ResponseContext?
The result of request will send to new created ResponseContext instead this object, 
You intend it?

Reply:
corrected to 
```csharp
this
```